build-ci-wheels/action.yml

```
@@ -0,0 +1,77 @@
+name: "Build a c-extension Python library"
+description: "Build a c-extension Python library by generating manylinux wheels"
+
+inputs:
+  CPython:
+    description: 'Desired CPython version'
+    default: '3.10'
+    required: false
+    type: string
+  cibw_build:
+    description: 'Desired build wheel cpython version'
+    default: "cp310-*"
+    required: false
+    type: string
+  cibw_skip:
+    description: 'Desired build wheel for skipping'
+    default: "*-musllinux* "
+    required: false
+    type: string
+  cibw_archs:
+    description: 'Desired build arhitecture'
+    default: " auto "
+    required: false
+    type: string
+  requires-pypy:
+      description: 'Whether requires build wheel for PyPy'
+      default: false
+      required: false
+      type: boolean
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: "Install Git and clone project"
+      uses: actions/checkout@v3
+   
+    - name: Set up QEMU
+      if: ${{ inputs.cibw_archs }} == 'aarch64'
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: arm64
+
+    - name: Build wheels for CPython ${{ inputs.python-version }}
+      if: ${{ inputs.cibw_build  != 'cp38-*'}} && ${{ inputs.requires-pypy }} == 'false' 
```

Conditional syntax in GitHub actions is tricky, as everything gets converted into a string. The following shows how to fix this line:

```suggestion
      if: ${{ inputs.cibw_build  != 'cp38-*'}} && ${{ inputs.requires-pypy == 'false' }}
```

Please, check the rest of the conditionals.